znft/keystore.go

```diff
@@ -0,0 +1,148 @@
+package znft
+
+import (
+	"fmt"
+	"os"
+	"path"
+
+	hdw "github.com/0chain/gosdk/zcncore/ethhdwallet"
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+)
+
+// ListStorageAccounts List available accounts
+func ListStorageAccounts(homedir string) []common.Address {
+	keyDir := path.Join(homedir, WalletDir)
+	ks := keystore.NewKeyStore(keyDir, keystore.StandardScryptN, keystore.StandardScryptP)
+	config := &accounts.Config{InsecureUnlockAllowed: false}
+	am := accounts.NewManager(config, ks)
```

Comment:
next func ``DeleteAccount`` uses the same 4 lines of code, can move it to another function or pass reference to Manager, make it struct instance probably

Reply:
Refactored